### DISCUSSION TITLE
Navigation outside of a step component?

### DISCUSSION THREAD

**Discussion by thedangler** [spatie/laravel-livewire-wizard#60] [Q&A]
Hello, 
Cool package.
I want to have a side bar navigation that is independent of each step component.
How can I access the steps when clicking on licks outside of the step component.

**Comment by ju5t** [comment]
Can you visualise this in some way? Because I either don't understand what you mean or I'm oversimplifying it. Because there is nothing holding you back from having a sidebar next to each component.

**Reply by thedangler** [reply]
I have a left nav bar that has my steps and a main container that has wizard that draws the steps
I've only been able to access wire:nextStep if  wire:nextStep is inside the step component blade file.

<nav>
<button wire:nextStep>Next</button>  // doesn work  
my link 1
my link to step 2  
</nav>
<div>
   <wizard>
       <step 1 component>
          stuff
          <button wire:nextStep>next</button>  // this works
      </step 1 compent>
   </wizard>
</div>


Its because the link in the nav is outside of the wizard component.
Come to think of it, i'll have to use events?  Unless you have another way?
My livewire skills are fresh.

**Reply by ju5t** [reply]
The navigation must be in a step component to work. There's some documentation on it here:
https://spatie.be/docs/laravel-livewire-wizard/v1/usage/rendering-navigation

If you don't want to repeat yourself in every blade file, you can of course use an include or create your own blade component.
